docs/sqlserver.md

```diff
@@ -0,0 +1,55 @@
+# `sqlserver` Metrics Receiver
+
+The `sqlserver` metrics receiver can fetch stats from a SQL Server via the [Windows Performance Counters](https://docs.microsoft.com/en-us/windows/win32/perfctrs/performance-counters-portal).
+
+## Prerequisites
+Because this `sqlserver` receiver makes use of the windows perforamnce counters, it is required for it to run locally on the same machine with the SQL Server installation.
+
+## Configuration
+
+| Field                 | Default            | Description |
+| ---                   | ---                | ---         |
+| `type`                | required           | Must be `sqlserver`. |
+| `collection_interval` | `60s`              | A [time.Duration](https://pkg.go.dev/time#ParseDuration) value, such as `30s` or `5m`. |
+
+Example Configuration:
+
+```yaml
+metrics:
+  receivers:
+    sqlserver:
+      type: sqlserver
+      collection_interval: 60s
+  service:
+    pipelines:
+      sqlserver:
+        receivers:
+          - sqlserver
+```
+
+## Metrics
+The Ops Agent collects the following metrics from SQL Server.
+Note: In order for any metrics to be retrieved, the receiver must run locally on the same server as the SQL Server installation.
+
+| Metric                                               | Data Type      | Unit        | Labels      | Description |
+| ---                                                  | ---            | ---         | ---         | ---         | 
+| workload.googleapis.com/sqlserver.user.connection.count | Gauge       | connections |             | Number of users connected to the SQL Server. |
+| workload.googleapis.com/sqlserver.lock.wait_time.avg | Gauge          | ms          |             | Average wait time for all lock requests that had to wait. |
+| workload.googleapis.com/sqlserver.lock.wait.rate     | Gauge          | requests/s  |             | Number of lock requests resulting in a wait. |
+| workload.googleapis.com/sqlserver.batch.request.rate | Gauge          | requests/s  |             | Number of batch requests received by SQL Server. |
+| workload.googleapis.com/sqlserver.batch.sql_compilation.rate | Gauge  | compilations/s |          | Number of SQL compilations needed. |
+| workload.googleapis.com/sqlserver.batch.sql_recompilation.rate | Gauge | compilations/s |         | Number of SQL recompilations needed. |
+| workload.googleapis.com/sqlserver.page.buffer_cache.hit_ratio | Gauge | %           |             | Pages found in the buffer pool without having to read from disk. |
```

Comment:
`Percent of pages` or `Fraction of pages`?

Reply:
changed